README.md

```diff
@@ -18,2309 +18,93 @@ The Ethereum connector allows you to access the Ethereum JSON RPC API through Ba
-{
-  "id":1999,
-  "jsonrpc":"2.0",
-  "result": "0xcce8842bc862eec936b6ca0c392c8633"
-}
-```
-##### Quick Testing
-`bin$ ballerina run sample.bal "http://localhost:8080" "2.0" 1999 eth_newFilter '{"fromBlock": "0x1","toBlock": "0x2","address": "0x0eb8a07d29f5afdcbec1a9d087ece456139bfb87","topics":["0x000000000000000000000000a94f5374fce5edbc8e2a8697c15331677e6ebf0b", null,["0x000000000000000000000000a94f5374fce5edbc8e2a8697c15331677e6ebf0b", "0x0000000000000000000000000aff3454fce5edbc8cca8697c15331677e6ebccc"]]}'`
-
-
-##### Related Ethereum Documentation
-
-https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_newfilter
-
-***
-
-#### ethNewBlockFilter
-
-Creates a filter in the node, to notify when a new block arrives.
-To check if the state has changed, call [ethGetFilterChanges](#ethgetfilterchanges).
-
-##### Parameters
-1. `string` - JSON RPC version
-2. `int` - network id
-
-##### Returns
-
-- A filter id.
-
-##### Example
-
-```ballerina
-//code
-json response = {};
-http:HttpConnectorError e;
-response, e = ethereumConnector.ethNewBlockFilter("2.0", 1999);
-if (e == null) {
-    println(response);
-} else {
-    println(e);
-}
-```
-
-```json
-{
-  "id":1999,
-  "jsonrpc":"2.0",
-  "result": "0xd1e74a0ecefd34817099171b8b9e8a82"
-}
-```
-##### Quick Testing
-`bin$ ballerina run sample.bal "http://localhost:8080" "2.0" 1999 eth_newBlockFilter`
-
-##### Related Ethereum Documentation
-
-https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_newblockfilter
-
-***
-
-#### ethNewPendingTransactionFilter
-
-Creates a filter in the node, to notify when new pending transactions arrive.
-
-##### Parameters
-1. `string` - JSON RPC version
-2. `int` - network id
-
-##### Returns
-
-- A filter id.
-
-##### Example
-
-```ballerina
-//code
-json response = {};
-http:HttpConnectorError e;
-response, e = ethereumConnector.ethNewPendingTransactionFilter("2.0", 1999);
-if (e == null) {
-    println(response);
-} else {
-    println(e);
-}
-```
-
-```json
-{
-  "id":1999,
-  "jsonrpc":"2.0",
-  "result": "0x1bad3335bdd994b7a4690f0a134fb340"
-}
-```
-##### Quick Testing
-`bin$ ballerina run sample.bal "http://localhost:8080" "2.0" 1999 eth_newPendingTransactionFilter`
-
-##### Related Ethereum Documentation
-
-https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_newpendingtransactionfilter
-***
-
-#### ethUninstallFilter
-
-Uninstalls a filter with given id.
-
-
-##### Parameters
-
-1. `string` - JSON RPC version
-2. `int` - network id
-3. `string` - The filter id.
-
-##### Returns
+5) Start the miner in the Geth JavaScript console using the following command.
+    ```
+    miner.start()
+    ```
+    
+The following sections provide you with information on how to use the Ballerina Ethereum connector.
 
-- `true` if the filter was successfully uninstalled, otherwise `false`.
+- [Contribute To Develop](#contribute-to-develop)
+- [Working with Ethereum Connector](#working-with-ethereum-connector)
+- [Sample](#sample)
 
-##### Example
+### Contribute To develop
```

Comment:
Use title case. Apply this everywhere. 

Also, this should be renamed to "Contribute to the Code" or something like that.